app/src/main/java/com/vitorpamplona/amethyst/ui/qrcode/QrCodeScanner.kt

```diff
@@ -48,6 +49,7 @@ fun QrCodeScanner(onScan: (String?) -> Unit) {
         setPrompt(stringResource(id = R.string.point_to_the_qr_code))
         setBeepEnabled(false)
         setOrientationLocked(false)
+        addExtra(Intents.Scan.SCAN_TYPE, Intents.Scan.MIXED_SCAN)

```

Comment:
That was it. Works fine now for all kinds of QR codes that I could test (inverted white-black and colors)